app-android/app/src/main/java/de/tutao/tutanota/Utils.kt

```diff
@@ -138,3 +139,10 @@ fun parseColor(color: String): Int {
 
 fun HttpURLConnection.iterateDataAsLines(action: (line: String) -> Unit) =
 		BufferedReader(InputStreamReader(BufferedInputStream(this.inputStream))).forEachLine(action)
+
+val Int.toPx: Int
```

Comment:
I wouldn't make it a property because it even has `to` in name. It can be an extension but it probably makes most sense as freestanding function like `dpToPx()`?

Reply:
You're right :+1: , I think I will create an extension.